QueueAndStack/Program.cs

```diff
@@ -0,0 +1,101 @@
+﻿namespace QueueAndStack
+{
+    public class Program
+    {
+        static void Main(string[] args)
+        {
+            // Task 1:
+            var newQueue = AddValuesOfTheQueue();
+            Console.WriteLine($"The queue with the deleted element {DeleteValueInTheQueue(newQueue)}");
```

Comment:
The output in the console is the type of the object

Reply:
😞 Fixed.